draft-welzl-iccrg-pacing.md

```diff
@@ -70,11 +70,43 @@ Because of its known advantages, pacing has become common in implementations of
 
 # Pacing: general considerations and consequences
 
-TODO
+## More likely to saturate a bottleneck {#losstypes}
+
+We can distinguish between two reasons for packet losses that are due to congestion at a bottleneck with a DropTail (FIFO) queue:
```

Comment:
Is it also productive later to also consider the case of AQM?  I think we would agree that pacing has benefits when paired with AQM as well, not just in the FIFO case.

Reply:
AQM is more random, and this is me paraphrasing my model, which is based upon a FIFO queue. I agree that saying something about AQM would be good, but... perhaps further down, as a separate subsection?